### ISSUE TITLE
Reword section in EBF-Time quest to reduce confusion.

### ISSUE THREAD

**Issue by querns** [GTNewHorizons/GT-New-Horizons-Modpack#9497]
### Your GTNH Discord Username

querns#3785

### Your Pack Version

2.1.2.1

### Your Proposal

Change the quest text of the EBF-Time (ID 77) to remove the reference to an LV EBF configuration.

### Your Goal

The current EBF-time quest has a section that says the following:

>The EBF also has a heat adjustment of 100K for each power tier +/- MV. Basically, for CuNI Coils, with an LV energy hatch, the heat capacity is 1701K (-100), MV=1801K (no change), HV=1901K (+100), etc. Coils only set the base value.

This is somewhat confusing, because it can read as if a standard configuration with two LV hatches actually drops the heat down -100K.

### Your Vision

I suggest rewording the section to the following:

>The EBF also has a heat adjustment of 100K for each power tier above or below MV. You can only make LV energy hatches right now, but installing two LV energy hatches will overclock the EBF to MV power, eliminating the 100K penalty and setting the initial heat to 1801K.

>As you upgrade your EBF over the course of the game, this heat adjustment will increase. Two MV energy hatches (overclocking to HV) will add +100K, two HV energy hatches (overclocking to EV) will give +200K, and so on. Coils only set the base value.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by miozune** [comment]
Fixed by #9496
